src/Regex.php

```diff
@@ -20,19 +22,23 @@ public static function match(string $pattern, string $subject): MatchResult
      * @param string $subject
      *
      * @return \Spatie\Regex\MatchAllResult
+     *
+     * @throws RegexFailed
      */
     public static function matchAll(string $pattern, string $subject): MatchAllResult
     {
         return MatchAllResult::for($pattern, $subject);
     }
 
     /**
-     * @param string|array $pattern
+     * @param string|array          $pattern
      * @param string|array|callable $replacement
-     * @param string|array $subject
-     * @param int $limit
+     * @param string|array          $subject
+     * @param int                   $limit
      *
      * @return \Spatie\Regex\ReplaceResult
+     *
+     * @throws RegexFailed
```

Comment:
Please revert the `@throws` tag additions, as we don't maintain those in our packages